src/components/newProposal/actions/ChangeParamInput.tsx

```diff
@@ -174,6 +211,8 @@ export const ChangeParamInput = () => {
         />
       }
     >
+      {variablesLoading && 'Retrieving plugins from DAO'}
+      {variablesErrors && 'Could not retrieve plugins from DAO'}
```

Comment:
I will add a retry button when there's an error, if you could be so kind as to add the onClick event for it